### PR TITLE
Focus management

### PR DESCRIPTION
**Pull request by hollsk**
Adds two sections about programmatic focus. Hopefully self-explanatory.

### CODE REVIEW

File: accessibility/common-remediations.md [hunk 1]
```diff
@@ -690,3 +692,60 @@ We _don’t_ do this:
...
+#### Who's affected by the problem?
+
+* Screen readers
+* Keyboard users
+
+#### Why's it a problem?
+
+"Focus management" is the practice of setting the user's current focus programmatically. 
+
+Most SPA frameworks fail to handle this automatically, and you need to implement it yourself. When a user activates a control to trigger a route change, the view visually updates. When you don't implement focus management, screen reader users are not made aware that the context has changed, and are left guessing whether or not activating the control did anything. Keyboard users may be able to see the visual changes, but they're left with their keyboard focus still on the trigger control. The control is no longer visible on the screen, and may require additional, unnecessary work from the keyboard user to get reoriented.
```
Comment (sky-jack) [inline accessibility/common-remediations.md]: Should this comma be removed? as it is followed by 'and'.

File: accessibility/common-remediations.md [hunk 2]
```diff
@@ -690,3 +692,60 @@ We _don’t_ do this:
...
+* Screen readers
+* Keyboard users
+
+#### Why's it a problem?
+
+"Focus management" is the practice of setting the user's current focus programmatically. 
+
+Most SPA frameworks fail to handle this automatically, and you need to implement it yourself. When a user activates a control to trigger a route change, the view visually updates. When you don't implement focus management, screen reader users are not made aware that the context has changed, and are left guessing whether or not activating the control did anything. Keyboard users may be able to see the visual changes, but they're left with their keyboard focus still on the trigger control. The control is no longer visible on the screen, and may require additional, unnecessary work from the keyboard user to get reoriented. 
+
+Implementing ARIA patterns that require focus management (e.g. modal dialogs), but neglecting to implement that focus management, can also confuse users or prevent them from being able to use your document.
```
Comment (sky-jack) [inline accessibility/common-remediations.md]: I'd say this comma is unnecessary.

File: accessibility/common-remediations.md (hunk 1 above)
Comment (hollsk) [reply accessibility/common-remediations.md]:
```suggestion
Most SPA frameworks fail to handle this automatically, and you need to implement it yourself. When a user activates a control to trigger a route change, the view visually updates. When you don't implement focus management, screen reader users are not made aware that the context has changed, and are left guessing whether or not activating the control did anything. Keyboard users may be able to see the visual changes, but they're left with their keyboard focus still on the trigger control. The control is no longer visible on the screen and may require additional, unnecessary work from the keyboard user to get reoriented. 
```

File: accessibility/common-remediations.md (hunk 2 above)
Comment (hollsk) [reply accessibility/common-remediations.md]:
```suggestion
Implementing ARIA patterns that require focus management (e.g. modal dialogs), but neglecting to implement that focus management can also confuse users or prevent them from being able to use your document.  
```

**Comment by hollsk** [comment]
Thanks! I take out entire clauses sometimes and forget to remove, the comma at the same, time

(i wiped out your approval tick making the changes, tho)

**Pull request merged**